kolibri_instant_schools_plugin/assets/src/views/profile-page/index.vue

```diff
@@ -177,52 +185,41 @@
         return null;
       },
       permissionTypeText() {
-        if (this.permissionType === PermissionTypes.SUPERUSER) {
-          return this.$tr('isSuperuser');
-        } else if (this.permissionType === PermissionTypes.LIMITED_PERMISSIONS) {
-          return this.$tr('limitedPermissions');
+        switch (this.permissionType) {
+          case PermissionTypes.SUPERUSER:
+            return this.$tr('isSuperuser');
+          case PermissionTypes.LIMITED_PERMISSIONS:
+            return this.$tr('limitedPermissions');
+          default:
+            return '';
         }
-        return '';
-      },
-      canEditUsername() {
-        if (this.isCoach || this.isLearner) {
-          return this.facilityConfig.learnerCanEditUsername;
-        }
-        return true;
       },
       canEditName() {
-        if (this.isCoach || this.isLearner) {
-          return this.facilityConfig.learnerCanEditName;
-        }
-        return true;
+        return this.userIsAdmin || this.facilityConfig.learnerCanEditName;
       },
-      nameIsInvalidText() {
-        if (this.nameBlurred || this.formSubmitted) {
-          if (this.name === '') {
-            return this.$tr('required');
-          }
-        }
-        return '';
+      canEditPassword() {
+        return this.userIsAdmin || this.facilityConfig.learnerCanEditPassword;
```

Comment:
So superuser learners and coaches cannot edit their password?

Reply:
As there should not be too many superusers floating about in the VF use case, we can probably defer that question for consideration on Kolibri - open an issue on Kolibri about it?

Reply:
This page might not have been updated after the superuser permissions stuff. To me, `userIsAdmin` might use the `isAdmin` getter, which looks for the superuser role.